lib/pbench/test/unit/agent/test_utils.py

```diff
@@ -29,20 +29,24 @@ def test_constructor(self):
             bs = OurServer(arg1, arg2)
             assert bs.pid_file is None
             assert bs.host == "localhost"
-            assert bs.port == 4242
+            assert bs.port == OurServer.def_port
             assert bs.bind_host == bs.host
             assert bs.bind_port == bs.port
-            assert repr(bs) == "forty-two - localhost:4242"
+            assert repr(bs) == f"forty-two - localhost:{OurServer.def_port:d}"
 
         _def = "localhost"
 
         with pytest.raises(OurServer.Err) as excinfo:
             bs = OurServer("bad_host.example.com", _def)
-        assert excinfo.value.return_code == 43, f"{excinfo.value!r}"
+        assert (
+            excinfo.value.return_code == OurServer.bad_host_ret_code
+        ), f"{excinfo.value!r}"
```

Comment:
I hate to ask, but...does `excinfo` have a `value` attribute?  If so, what is it?

(And, is `f"{excinfo.value!r}"` better than `repr(excinfo.value)`?)

Ditto line 49.

Reply:
`value` is the exception object that was raised.  Go figure.